msg/GenericTrajectory.msg

```diff
@@ -0,0 +1,3 @@
+Header header
+trajectory_msgs/JointTrajectory joint_trajectory
```

Comment:
why allow for multiple Cartesian trajectories, but only one joint trajectory?

Reply:
See above
https://github.com/ros-planning/moveit_msgs/pull/46/files#r328268508

Reply:
In addition, this allows you to have some separate semantic distinctions between paths (i.e., all of trajectories in the Generic trajectory go to the same goal, and can be used in combination during whatever optimization step happens), whereas each Generic trajectory is a separate target.

Reply:
And that is what I'm a little worried about. Maybe this is something we should mention, as having a vector of 2 vectors can quickly become unclear.

Reply:
I added the vectorization to GenericTrajectory and a comment on semantic equivalence